manage.py

```diff
@@ -1,12 +1,38 @@
 from flask.ext.script import Manager
 from flask.ext.migrate import Migrate, MigrateCommand
-from config import SQLALCHEMY_DATABASE_URI
+from config import SAMPLE_DATABASE_URI
 from app import application, db
+import sqlite3, sys
  
 migrate = Migrate(application, db)
  
 manager = Manager(application)
 manager.add_command('db', MigrateCommand)
+
+@manager.command
+def import_db():
+    try:
+        con = sqlite3.connect(SAMPLE_DATABASE_URI)
+    except:
+        print("\nCould not connect to database")
+        sys.exit(0)
+    cursor = con.execute("SELECT * from user_system_info limit 1")
+    result= cursor.fetchone()
+    if result:
+        choice = input("Database already contains some data. Do you wish to delete all data and overwrite? (y/n) :")
```

Comment:
Instead of query the user, could you request to run the command with `--overwrite`?